documents/motions/02.2022.1 - Definitions.md

```diff
@@ -52,15 +52,11 @@ Within WCA we use the following definitions for terminology.
     </tr>
     <tr>
       <td>Community</td>
-      <td>The Community of the WCA as formed by all Staff, Registered Speedcubers, and Partners.</td>
+      <td>The Community of the WCA as formed by all WCA Staff, Registered Speedcubers, and Partners.</td>
     </tr>
     <tr>
       <td>Competition Organizer</td>
-      <td>Persons who are responsible for the organization of a WCA Competition, as defined in the WCA Regulations.</td>
-    </tr>
-    <tr>
-      <td>Competition Staff</td>
-      <td>Persons who are responsible for the operational procedures during a WCA Competition, as defined in the WCA Regulations.</td>
```

Comment:
If we wanted to keep this role in the definitions we can but would need to change it to "Officials" or "Competition Officials" to match the wording of regulations article 1